### ISSUE TITLE
Feature request: Configurable date view for ativities inside card

### ISSUE THREAD

**Issue by zombah** [wekan/wekan#1673]
## Feature request

Now to find exact date of activity inside card you need mouseover relative date, like month ago, of activity and wait for tooltip with exact date to appear, but it is not possible to copy it for example.

It would be nice to have option to switch date view between relative and exact date for activities per card, board or even global per instance.

**Comment by TiibCD** [comment]
I agree it would be nice to have the exact date instead of the relative date (which for me is useless)
